staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -975,9 +975,9 @@ type PodSecurityPolicySpec struct {
 	// +optional
 	DefaultAllowPrivilegeEscalation *bool `json:"defaultAllowPrivilegeEscalation,omitempty" protobuf:"varint,15,opt,name=defaultAllowPrivilegeEscalation"`
 	// AllowPrivilegeEscalation determines if a pod can request to allow
-	// privilege escalation.
+	// privilege escalation. If unspecified, defaults to true.
```

Comment:
Is this an API change? undefined was leading to `false` before.

Moreover, I expected to be safe by default, i.e. default to false.

Reply:
Prior to 1.8, `AllowPrivilegeEscalation` was implicitly always true. But I think this needs to be true to fix https://github.com/kubernetes/kubernetes/issues/53437, which motivated this change.

Reply:
> Is this an API change? undefined was leading to false before.

Correct, which was a breaking change in 1.8.0

> Moreover, I expected to be safe by default, i.e. default to false.

Compatibility wins. The ability to restrict this did not exist prior to 1.8.0, so policies that do not include this field must continue pre-1.8.0 behavior.